pkgs/tools/misc/betterdiscordctl/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, pkgs }:
+
+stdenv.mkDerivation rec {
+  pname = "betterdiscordctl";
+  version = "1.6.1";
```

Comment:
bb010g/betterdiscordctl#56 was merged.
```suggestion
  version = "1.7.0";
```

Reply:
`fetchFromGitHub` fetches releases here, and 1.7.0 does not seem to be released yet. It's preffered to use a stable release, rather than a git rev which we would need to do to include that.

Reply:
I pushed the release.